src/saving/InProgressSave.cs

```diff
@@ -30,8 +30,14 @@ public class InProgressSave : IDisposable
 
     private bool success;
     private string message;
+
+    /// <summary>
+    ///   Failure exception or description. Depends on exceptionIsException
+    /// </summary>
     private string exception;
 
+    private bool exceptionIsException = true;
```

Comment:
That variable sounds very... strange. Maybe rename the `exception` variable?

Reply:
I did try, couldn't really think of anything better than what I ended up with.